alpha/lib/model/LiveEntry.php

```diff
@@ -121,7 +123,22 @@ public function generateFileName( $sub_type, $version = null)
 		return parent::generateFileName($sub_type, $version);
 	}
 	
-	protected $decidingLiveProfile = false;
+	/**
+	 * Code to be run before updating the object in database
+	 * @param PropelPDO $con
+	 * @return bloolean
+	 */
+	public function preUpdate(PropelPDO $con = null)
+	{
+		if($this->isColumnModified(entryPeer::CONVERSION_PROFILE_ID))
```

Comment:
I think that if recording status or options changed, you also should clear the recorded and redirected ids.